src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -61,305 +44,27 @@ export const ShippingRoute: FC<ShippingProps> = props => {
         order={props.order}
         me={props.me}
       >
-        <ShippingRouteLayout
-          dialog={props.dialog}
-          order={props.order}
-          me={props.me}
-          router={props.router}
-        />
+        <ShippingRouteLayout order={props.order} me={props.me} />
       </ShippingContextProvider>
     </Analytics>
   )
 }
 
-const ShippingRouteLayout: FC<ShippingProps> = props => {
-  const { order } = props
-
-  const createSavedAddress = useCreateSavedAddress()
-  const updateSavedAddress = useUpdateSavedAddress()
-  const deleteSavedAddress = useDeleteSavedAddress()
-  const selectShippingQuote = useSelectShippingQuote()
-
+const ShippingRouteLayout: FC<Omit<ShippingProps, "dialog">> = ({
```

Comment:
As with the context, with everything strictly co-located this component is now very slim and specific.